pkgs/by-name/pa/parsify/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+,appimageTools
+,fetchurl
+}:
+
+appimageTools.wrapType2 rec {
+  pname = "parsify";
+  version = "2.0.1";
+
+  src = fetchurl {
+    url = "https://github.com/parsify-dev/desktop/releases/download/v${version}/Parsify-${version}-linux-x86_64.AppImage";
+    hash = "sha256-ltWqRW+cBvuUJzhya62WsBY5zqIua9xhilxfd9gr24A=";
+  };
+
+  extraInstallCommands = ''
+    mv $out/bin/${pname}-${version} $out/bin/${pname}
+  '';
+
+  meta = with lib; {
+    description = "Next generation notepad-based calculator, built with extendibility and privacy in mind.";
+    homepage = "https://parsify.app/";
+    license = licenses.unfree;
+    maintainers = with maintainers; [ kashw2 ];
+    platforms = platforms.linux;
```

Comment:
```suggestion
    platforms = platforms.linux;
    mainProgram = "parsify";
```

Reply:
Thanks for the catch, amended in https://github.com/NixOS/nixpkgs/pull/274845/commits/b5162bfa84e012181547a8dfc0ff12b27a20aec9